src/main/cli/settings.c

```diff
@@ -1819,12 +1814,14 @@ const clivalue_t valueTable[] = {
     { PARAM_NAME_ALTITUDE_D_LPF,        VAR_UINT16 | MASTER_VALUE, .config.minmaxUnsigned = { 10, 1000 }, PG_POSITION, offsetof(positionConfig_t, altitude_d_lpf) },
 
 // PG_POSITION_CONTROL
-    { PARAM_NAME_HOVER_THROTTLE,        VAR_UINT16 | MASTER_VALUE, .config.minmaxUnsigned = { 1100, 1700 }, PG_POSITION_CONTROL, offsetof(positionControlConfig_t, hover_throttle) },
-    { PARAM_NAME_LANDING_ALTITUDE,      VAR_UINT8  | MASTER_VALUE, .config.minmaxUnsigned = { 0, 200 }, PG_POSITION_CONTROL, offsetof(positionControlConfig_t, landing_altitude_m) },
-    { PARAM_NAME_ALTITUDE_P,            VAR_UINT8  | MASTER_VALUE, .config.minmaxUnsigned = { 0, 200 }, PG_POSITION_CONTROL, offsetof(positionControlConfig_t, altitude_P) },
-    { PARAM_NAME_ALTITUDE_I,            VAR_UINT8  | MASTER_VALUE, .config.minmaxUnsigned = { 0, 200 }, PG_POSITION_CONTROL, offsetof(positionControlConfig_t, altitude_I) },
-    { PARAM_NAME_ALTITUDE_D,            VAR_UINT8  | MASTER_VALUE, .config.minmaxUnsigned = { 0, 200 }, PG_POSITION_CONTROL, offsetof(positionControlConfig_t, altitude_D) },
-    { PARAM_NAME_ALTITUDE_F,            VAR_UINT8  | MASTER_VALUE, .config.minmaxUnsigned = { 0, 200 }, PG_POSITION_CONTROL, offsetof(positionControlConfig_t, altitude_F) },
+    { PARAM_NAME_HOVER_THROTTLE,           VAR_UINT16 | MASTER_VALUE, .config.minmaxUnsigned = { 1100, 1700 }, PG_POSITION_CONTROL, offsetof(positionControlConfig_t, hover_throttle) },
+    { PARAM_NAME_ALT_CONTROL_THROTTLE_MIN, VAR_UINT16 | MASTER_VALUE, .config.minmaxUnsigned = { 1050, 1400 }, PG_POSITION_CONTROL, offsetof(positionControlConfig_t, alt_control_throttle_min) },
+    { PARAM_NAME_ALT_CONTROL_THROTTLE_MAX, VAR_UINT16 | MASTER_VALUE, .config.minmaxUnsigned = { 1400, 2000 }, PG_POSITION_CONTROL, offsetof(positionControlConfig_t, alt_control_throttle_max) },
```

Comment:
The limits seem arbitrary. Why not let pilots choose throttle values within the possible range of 1000-2000?

Reply:
We had issues in GPS Rescue where pilots set the max throttle too low leading to a crash due to inability to maintain altitude.  The parameter is almost of no relevance, it's only useful function is to avoid excessive motor drive if the quad gets stuck in a tree on the way home during a GPS Rescue.  Then it limits the max that iTerm can push the motors to.  But in that case, things don't look good anyway.
Also, it must be set more than the min value. 2000 would be below min.  And then zero throttle control even if we set them the same.  
These limits were put there 'historically' in the old GPS Rescue code.  It would probably better to remove these user configurable limits, and set the minimum to the idle value and maximum to the value that the user's current throttle scaling uses.
But that's really for a separate PR.
This is a refactoring PR.